js/packages/logger/src/handler-rolling-file.ts

```diff
@@ -65,6 +65,15 @@ function ensureDirectoryExistence(filename: string) {
   const dirname = path.dirname(filename)
   if (!fs.existsSync(dirname)) {
     ensureDirectoryExistence(dirname)
-    fs.mkdirSync(dirname)
+    try {
+      fs.mkdirSync(dirname)
+    } catch (ex) {
+      if (ex.message.includes('file already exists')) {
+        // another process already created this file
+        // do nothing
+      } else {
+        throw ex
+      }
```

Comment:
I believe an empty `if` block is a bad practice 
```suggestion
      // if a file already exists ignore the error
      if (!ex.message.includes('file already exists')) {
        throw ex
      }
```